src/client/components/Modal/Modal.styles.css

```diff
@@ -0,0 +1,23 @@
+* {
+  text-align: center;
+}
+
+.btnCart {
```

Comment:
When you use the button component, then you can get rid of this

Reply:
thanks.. but i need these ..so this component is independent of it..